markets/legacy-market/contracts/LegacyMarket.sol

```diff
@@ -277,16 +284,38 @@ contract LegacyMarket is ILegacyMarket, Ownable, UUPSImplementation, IMarket, IE
             debtValueMigrated
         );
 
+        if (v3System.isVaultLiquidatable(preferredPoolId, address(oldSynthetix))) {
+            revert Paused();
+        }
+
         // send the built v3 account to the staker
-        IERC721(v3System.getAccountTokenAddress()).safeTransferFrom(
-            address(this),
-            staker,
-            accountId
-        );
+        try
+            IERC721(v3System.getAccountTokenAddress()).safeTransferFrom(
+                address(this),
+                staker,
+                accountId
+            )
+        {} catch {
```

Comment:
its not necessary to filter by specific revert reason because at this point any case where we don't actually conclude migrting the user's account is a case where we could potentially be unable to liquidate a user. there are a great many errors that could be returned here outside the one that we are expecting, but the idea is if any error *does* occur, we will allow for the account to be deferred-transferred in those extreme cases.